scm-ui/ui-webapp/src/repos/components/form/RepositoryFormSwitcher.tsx

```diff
@@ -21,22 +21,12 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
  * SOFTWARE.
  */
-
 import React, { FC } from "react";
-import styled from "styled-components";
-import { Button, ButtonAddons, Icon, Level, urls } from "@scm-manager/ui-components";
 import { useLocation } from "react-router-dom";
 import { useTranslation } from "react-i18next";
-
-const MarginIcon = styled(Icon)`
-  padding-right: 0.5rem;
-`;
-
-const SmallButton = styled(Button)`
-  border-radius: 4px;
-  font-size: 1rem;
-  font-weight: 600;
```

Comment:
What happened with this styling ? Do we need `has-text-weight-semibold` ?

Reply:
600 is still normal thickness.

Reply:
A button has these properties by default.